src/Functions/countSubstringsCaseInsensitiveUTF8.cpp

```diff
@@ -19,6 +19,41 @@ using FunctionCountSubstringsCaseInsensitiveUTF8 = FunctionsStringSearch<CountSu
 
 REGISTER_FUNCTION(CountSubstringsCaseInsensitiveUTF8)
 {
-    factory.registerFunction<FunctionCountSubstringsCaseInsensitiveUTF8>();
+    FunctionDocumentation::Description description = R"(
+Returns how often a substring `needle` occurs in a string `haystack`.
```

Comment:
Like countSubstrings but counts case-insensitively and assumes that haystack is a UTF-8 string.